<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nativeVideo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>384</width>
      <height>216</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="postproVideo">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>384</width>
      <height>216</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>startButton</tabstop>
  <tabstop>pauseButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>clickStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>clickExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>536</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clickStart()</slot>
  <slot>clickPause()</slot>
  <slot>clickSave()</slot>
  <slot>clickExit()</slot>
 </slots>
</ui>
